api/buildcraft/api/enums/EnumHandlerPriority.java

```diff
@@ -0,0 +1,17 @@
+/*
+ * Copyright (c) 2017 SpaceToad and the BuildCraft team
+ * This Source Code Form is subject to the terms of the Mozilla Public License, v. 2.0. If a copy of the MPL was not
+ * distributed with this file, You can obtain one at https://mozilla.org/MPL/2.0/
+ */
+
+package buildcraft.api.enums;
+
+public enum EnumHandlerPriority implements Comparable<EnumHandlerPriority> {
+    HIGHEST,
+    HIGH,
+    NORMAL,
+    LOW,
+    LOWEST;
+
+    public static final EnumHandlerPriority[] VALUES = values();
+}
```

Comment:
Please leave this class where it was, the enum package also needs api cleanup (as mentioned above)